### ISSUE TITLE
p-columnfilter: missing aria-label

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#14537]
The main ToggleButton (.p-column-filter-menu-button) still missing aria-label. Buttons inside filter menu (Apply & Clear) are ok now. Tested with primeNg 17.3.2: https://primeng.org/table#filter

_Originally posted by @TaneliTuomola in https://github.com/primefaces/primeng/issues/14272#issuecomment-1852113702_

**Comment by cetincakiroglu** [comment]
Thanks a lot for reporting the issue!

**Comment by TaneliTuomola** [comment]
The main ToggleButton still missing aria-label. [There is wrong translation keys set in api.](https://github.com/primefaces/primeng/commit/12ca541981cf7c1b2d0c09b85fea1eb4b66608ed#r137434755)
